### ISSUE TITLE
[Package Request] Ripgrep-all

### ISSUE THREAD

**Issue by kcubeterm** [termux/termux-packages#6209]
- [x] I have read [Termux policy on package requests](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#a-note-about-package-requests).

**Package description**

rga is a line-oriented search tool that allows you to look for a regex in a multitude of file types. rga wraps the awesome ripgrep and enables it to search in pdf, docx, sqlite, jpg, movie subtitles (mkv, mp4), etc.

In short: Advanced version of ripgrep.

**Link to home page and sources**
1. Home page: https://github.com/phiresky/ripgrep-all
2. Source code:https://github.com/phiresky/ripgrep-all

**Comment by xeffyr** [comment]
I'm getting
```
  Installing ripgrep_all v0.9.6 (/home/builder/.termux-build/ripgrep-all/src)
    Updating crates.io index
error: failed to compile `ripgrep_all v0.9.6 (/home/builder/.termux-build/ripgrep-all/src)`, intermediate artifacts can be found at `/home/builder/.termux-build/ripgrep-all/src/target`

Caused by:
  failed to select a version for the requirement `cachedir = "^0.1.1"`
  candidate versions found which didn't match: 0.3.0, 0.2.0
  location searched: crates.io index
  required by package `ripgrep_all v0.9.6 (/home/builder/.termux-build/ripgrep-all/src)`
```

**Comment by kcubeterm** [comment]
Unlike `cargo build`, `cargo install` ignore cargo.lock.
So have to use `--locked` options with `Cargo install` or `cargo build` is another choice

**Comment by kcubeterm** [comment]
Added a262445c2142b838d680c6cca43a43a574014152
